refs/heads/py3.9-call-atexit-despite-sigint
